index.bs

```diff
@@ -5796,12 +5789,12 @@ The following abstract operations operate on {{TransformStream}} instances at a
  id="transform-stream-error-writable-and-unblock-write">TransformStreamErrorWritableAndUnblockWrite(|stream|,
  |e|)</dfn> performs the following steps:
 
- 1. Perform ! |stream|.[=TransformStream/[[controller]]=].[=TransformStreamDefaultController/[[cancelAlgorithm]]=](|e|).
  1. Perform ! [$TransformStreamDefaultControllerClearAlgorithms$](|stream|.[=TransformStream/[[controller]]=]).
  1. Perform !
     [$WritableStreamDefaultControllerErrorIfNeeded$](|stream|.[=TransformStream/[[writable]]=].[=WritableStream/[[controller]]=], |e|).
  1. If |stream|.[=TransformStream/[[backpressure]]=] is true, perform ! [$TransformStreamSetBackpressure$](|stream|,
     false).
+ 1. Perform ! |stream|.[=TransformStream/[[controller]]=].[=TransformStreamDefaultController/[[finallyAlgorithm]]=]().
```

Comment:
This algorithm will already have been cleared by `TransformStreamDefaultControllerClearAlgorithms` in step 1, so this won't work.

Reply:
yea, it currently isnt, as i am unsure how to proceed without moving the clear algs